Dockerfile

```diff
@@ -113,17 +113,26 @@ COPY . /code/timemachine/
 WORKDIR /code/timemachine/
 RUN pip install --no-cache-dir -e . && rm -rf ./build
 
-# Container with only cuda runtime, half the size of the timemachine_cuda_dev container
-FROM nvidia/cuda:12.4.1-runtime-ubuntu20.04 as timemachine
+# Container with only cuda base, half the size of the timemachine_cuda_dev container
+# Need to copy curand/cudart as these are dependecies of the Timemachine GPU code
+FROM docker.io/nvidia/cuda:12.4.1-base-ubuntu20.04 as timemachine
 ARG LIBXRENDER_VERSION
 ARG LIBXEXT_VERSION
 RUN (apt-get update || true) && apt-get install --no-install-recommends -y libxrender1=${LIBXRENDER_VERSION} libxext-dev=${LIBXEXT_VERSION} \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
-COPY --from=timemachine_cuda_dev /opt/ /opt/
+# Copy all of the include directory, more headers than neccessary but headers are small
+COPY --from=timemachine_cuda_dev /usr/local/cuda/targets/x86_64-linux/include/ /usr/local/cuda/targets/x86_64-linux/include/
```

Comment:
Dumb question, do we actually need to copy any headers at this point (since I don't think we're going to build anything with this image)?

Reply:
Nope, will remove.

Reply:
Removed in https://github.com/proteneer/timemachine/pull/1435/commits/e691853a45d19b2c589f02002951a821efb6ff16